src/components/NcActionInput/NcActionInput.vue

```diff
@@ -190,15 +202,29 @@ export default {
 			return this.type === 'multiselect'
 		},
 
-		isDatePickerType() {
+		isNativeDatePickerType() {
```

Comment:
```suggestion
		nativeDatePickerType() {
```

`is*` implies a predicate that returns a boolean.

Reply:
ok, isDatePickerType was there before, and i've done a new computed prop also this way. I will correct it in both cases